README.md

```diff
@@ -15,7 +15,7 @@ engines:
     config:
       patterns:
         no-set-methods:
-          pattern: /def set_\w+/
+          pattern: "def set_\\w+"
```

Comment:
Following up on @pbrisbin's point - switching to a string is ok (and dropping the slashes). What would it take for the pattern to be `"def set \w+"`? Ideally, it should be the same as what you can pass to grep itself:

```
grep -E -e "def set_\w+" lib/cc/foo.rb
```